Gemfile

```
@@ -4,7 +4,7 @@ source "https://rubygems.org"
 #gem 'bugsnag-maze-runner', path: '../maze-runner'
 
 # Or a specific release:
-gem 'bugsnag-maze-runner', '~>9.13.0'
+gem 'bugsnag-maze-runner', '~>9.14.0'
```

Can we just take the latest v9?
```suggestion
gem 'bugsnag-maze-runner', '~>9.0'
```